ColorPatterns/Scenes/Game/GameCreator.swift

```diff
@@ -12,6 +12,7 @@ struct GameCreator {
 
     func getController(difficulty: Level) -> GameController {
 
+        // TODO: How about moving timer to viewModel? seems like it is not related to UI
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/Game/GameCreator.swift#L15"></span>TODOs should be resolved (How about moving timer to view…).<br />
<code>todo</code> <code>GameCreator.swift:15</code></td>
    </tr>
  </tbody>
</table>